static/components/ComponentArray.js

```diff
@@ -58,25 +58,21 @@ class ArrayComponent extends TextFieldComponent{
     var tpl = '';
     tpl += super.renderElement(textvalue,index);
     gArrayComponentId++;
-    tpl += `<div class='array-component-readonly-text' ref='readonly_display'></div>`
-    tpl += `<button class="btn btn-secondary btn-sm" type="button" data-toggle="collapse" data-target="#componentArrayCollapse${gArrayComponentId}" aria-expanded="false" aria-controls="collapseExample">Show Info</button>`
-    tpl += `<div class="collapse" id="componentArrayCollapse${gArrayComponentId}">`;
-    tpl += '<div class="d-sm-flex flex-row">';
-    tpl += '<div class="p-2">';
-    tpl += `<div>Data length: <span class='arrayComponentLength'></span></div>`;
-    tpl += `<div>Min: <span class='arrayComponentMin'></span></div>`;
-    tpl += `<div>Max: <span class='arrayComponentMax'></span></div>`;
-    tpl += `<div>Mean: <span class='arrayComponentMean'></span></div>`;
-    tpl += `<div>RMS: <span class='arrayComponentRMS'></span></div>`;
-    var bounds = this.getBounds();
-    if(!isNaN(bounds.hi)) tpl += `<div>Out-of-spec high: <span class='arrayComponentOobHi'>n/a</span></div>`;
-    if(!isNaN(bounds.lo)) tpl += `<div>Out-of-spec low: <span class='arrayComponentOobLo'>n/a</span></div>`;
-    tpl += "</div>"
-    tpl += `<div class='flex-grow-1 p-2 arrayComponentGraph' style='height:200px; width: 240px;'></div>`;
-    tpl += `<div class='flex-grow-1 p-2 arrayComponentHistogram' style='height:200px; width: 240px;'></div>`;
-
-    tpl += '</div>'
-    tpl += '</div>'
+//    tpl += `<div class='array-component-readonly-text' ref='readonly_display'></div>`
+      tpl += `<div class="d-flex justify-content-around border">`;
+        tpl += `<div class="align-self-center p-2">`;
+          tpl += `<div>Number of Wires: <span class='arrayComponentLength'></span></div>`;
+          tpl += `<div>Min. Tension (N): <span class='arrayComponentMin'></span></div>`;
+          tpl += `<div>Max. Tension (N): <span class='arrayComponentMax'></span></div>`;
+          tpl += `<div>Mean Tension (N): <span class='arrayComponentMean'></span></div>`;
+          tpl += `<div>RMS Tensions (N): <span class='arrayComponentRMS'></span></div>`;
+          var bounds = this.getBounds();
+          if(!isNaN(bounds.hi)) tpl += `<div>Out-of-spec high: <span class='arrayComponentOobHi'>n/a</span></div>`;
+          if(!isNaN(bounds.lo)) tpl += `<div>Out-of-spec low: <span class='arrayComponentOobLo'>n/a</span></div>`;
+        tpl += `</div>`
+        tpl += `<div class='flex-grow-1 p-4 arrayComponentGraph' style='height: 200px; width: 240px;'></div>`;
+        tpl += `<div class='flex-grow-1 p-4 arrayComponentHistogram' style='height: 200px; width: 240px;'></div>`;
+      tpl += `</div>`
```

Comment:
Consider using multiline string interpolation instead of concatenation for better readability:
```
tpl = `
  <div>Max: ... </div>
  <div>Mean: ... </div>
  ...
  <div>
    <span>Number of Wires...</span>
  </div>
`;
```

Reply:
Good point - was mainly just trying to keep as close to the original code as I could, but keep it difficult.

Reply:
I've changed as much as I could without breaking the formatting ... but I'm sure it can be improved further (I'm not yet so good with combining string interpolation and HTML tags).  Please let me know further changes.

Reply:
I cleaned up that file a bit more, stylistically.  See how the string interpolation can be accomplished with variables?  In the case of conditionals, ternary expressions can be used.
<img width="823" alt="image" src="https://user-images.githubusercontent.com/4521230/144663838-13ad4180-e113-4926-b57c-59d05be7383a.png">